dev-client/src/screens/ProjectTeamScreen/components/UserItem.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {Button, Center, Image, Pressable} from 'native-base';
+import {User} from 'terraso-client-shared/account/accountSlice';
+import {useTranslation} from 'react-i18next';
+import {useMemo} from 'react';
+import {ProjectMembership} from 'terraso-client-shared/project/projectSlice';
+import {ConfirmModal} from 'terraso-mobile-client/components/modals/ConfirmModal';
+import {formatName} from 'terraso-mobile-client/util';
+import {
+  Box,
+  HStack,
+  VStack,
+  Badge,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type TriggerProps = {
+  onOpen: () => void;
+  message: string;
+};
+
+function LeaveProjectTrigger({onOpen, message}: TriggerProps) {
+  return (
+    <Center>
+      <Button
+        size="sm"
+        my={2}
+        w="50%"
+        _text={{color: 'error.main'}}
+        bgColor="grey.200"
+        onPress={onOpen}>
+        {message}
+      </Button>
+    </Center>
+  );
+}
+
+type InfoProps = {
+  membership: ProjectMembership;
+  user: User;
+  isCurrentUser: boolean;
+};
+
+const UserInfo = ({membership, user, isCurrentUser}: InfoProps) => {
```

Comment:
question: this file defines two components: UserInfo and UserItem. Should they be in separate files?

Reply:
I can split them apart - this was already split away from the user list component.